.changeset/wicked-cheetahs-change.md

```diff
@@ -0,0 +1,28 @@
+---
+"@hashicorp/design-system-components": patch
+---
+
+Aligned private class properties to follow a standardized notation
+- `Accordion`
+- `Alert`
+- `AppHeader`
+- `AppSideNav`
+- `AppSideNav::List`
+- `AppSideNav::Portal`
+- `CodeBlock`
+- `Copy::Button`
+- `Copy::Snippet`
+- `DisclosurePrimitive`
+- `Dropdown`
+- `Flyout`
+- `Form::SuperSelect`
+- `Form::TextInput`
+- `Icon`
+- `Modal`
+- `Pagination::Compact`
+- `Pagination::Numbered`
+- `Pagination::SizeSelector`
```

Comment:
suggestion: same here, based on our documentation we don't expect `SizeSelector` to be used as standalone
```suggestion
- `Pagination::Compact`
- `Pagination::Numbered`
```